index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,700;1,500&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/97d9fb687f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" media="screen and (max-width: 765px)" href="style.css">
+    <title>Ayomide's portfolio</title>
+</head>
+<body>
+    <section>
+        <header class="header">
+            <nav class="navbar">
+                <div class="logo">
+                    <img src="./assets/My Logo.png" alt="">
+                </div>
+                <div class="hamburger">
+                    <span class="line-top"></span>
+                    <span class="line-middle"></span>
+                    <span class="line-bottom"></span>
+                </div>
+            </nav>
+            <div class="intro">
+                <h1 class="title">
+                    I'm Ayomide.<br>
+                    Glad to see you!
+                </h1>
+                <p class="bio">
+                    I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <h3 class="let-connect">
+                    LET'S CONNECT
+                </h3>
+                <div class="social-icons">
+                    <i class="fa-brands fa-twitter fa-lg"></i>
+                    <i class="fa-brands fa-linkedin fa-lg"></i>
+                    <i class="fa-solid fa-m fa-lg"></i>
+                    <i class="fa-brands fa-github fa-lg"></i>
+                    <i class="fa-regular fa-hand-peace fa-flip-horizontal fa-lg"></i>
+                </div>
```

Comment:
- [ ] As required for the project, kindly ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; and each icon has a link (`<a>` tag). Apart from being semantically correct, the <a> tag will let you connect your social networks with the icons, using `href` attribute.
```html
 <ul>
    <li>
      <a href="#"><i class="fa-brands fa-twitter fa-lg"></i></a>
    </li>
    <li>
      <a href="#"><i class="fa-brands fa-twitter fa-lg"></i></a>
    </li>
    <li>
      <a href="#"><i class="fa-brands fa-twitter fa-lg"></i></a>
    </li>
    ...
  </ul>
```
**Once you have fixed the above change, you should ensure that your icons still following the Figma design**